perception/sandbox/polygrasp/msetup.py

```diff
@@ -0,0 +1,33 @@
+import mrp
+
+mrp.process(
+    name="segmentation_server",
+    runtime=mrp.Conda(
+        run_command=["python", "-m", "utils.mrp_wrapper"],
+        use_named_env="unseen-object-clustering",
```

Comment:
Add a comment pointing to instructions for making the named env.

Also, please add a TODO to make this not use a predefined env.

Same below.